libraries/launcher/org/multimc/applet/LegacyFrame.java

```diff
@@ -0,0 +1,160 @@
+package org.multimc.applet;/*
+ * Copyright 2012-2021 MultiMC Contributors
+ *
```

Comment:
package should probably go below the license header

Reply:
Fixed